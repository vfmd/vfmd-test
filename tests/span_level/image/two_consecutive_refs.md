Intertwined ![one][two][three]

Intertwined ![one][four][three]

[two]: url2
[three]: url3
