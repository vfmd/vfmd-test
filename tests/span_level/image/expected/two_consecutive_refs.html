<p>Intertwined <img src="url2" alt="one" /><a href="url3">three</a></p>

<p>Intertwined ![one][four]<a href="url3">three</a></p>
